src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -89,21 +97,66 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
 
   const [selectedDayIndex, setSelectedDayIndex] = useState(0);
 
+  const { minDate, maxDate } = relatedVenueEvents.reduce(
+    (resultObj, event) => {
+      if (resultObj.minDate === 0 || resultObj.maxDate === 0) {
+        resultObj.minDate =
+          resultObj.minDate === 0 ? event.start_utc_seconds : resultObj.minDate;
+
+        const secondsLength = minutesToSeconds(event.duration_minutes);
+        resultObj.maxDate =
+          resultObj.maxDate === 0
+            ? event.start_utc_seconds + secondsLength
+            : resultObj.maxDate;
+      }
+
+      if (event.start_utc_seconds < resultObj.minDate) {
+        resultObj.minDate = event.start_utc_seconds;
+      }
+
+      if (
+        event.start_utc_seconds + minutesToSeconds(event.duration_minutes) >
+        resultObj.maxDate
+      ) {
+        const secondsLength = minutesToSeconds(event.duration_minutes);
+
+        resultObj.maxDate = event.start_utc_seconds + secondsLength;
+      }
+
+      return resultObj;
+    },
+    { minDate: 0, maxDate: 0 }
+  );
+  const milisecondStart = new Date(secondsToMilliseconds(minDate)).getTime();
+  const milisecondFinish = new Date(secondsToMilliseconds(maxDate)).getTime();
+  const dayDifference = Math.round(
+    millisecondsToHours(milisecondFinish - milisecondStart) / 24
+  );
```

Comment:
can we use `differenceInDays` to calculate the number of days and `fromUnixTime` to convert minDate and maxDate from seconds to the date object? 🙃 

so that it would look like:
```js
const dayDifference = differenceInDays(fromUnixTime(maxDate), fromUnixTime(minDate));
```